api/v1alpha/generated/saucectl.schema.json

```diff
@@ -43,6 +43,10 @@
                 "description": "Manage test output, such as logs, videos, and screenshots.",
                 "type": "object",
                 "properties": {
+                  "cleanupPrevious": {
+                    "description": "Whether to cleanup previous downloaded artifacts",
```

Comment:
Not entirely true. You are removing all contents of the artifact dir. If the user specifies a directory that has more than just saucectl downloaded artifacts, it will delete everything at the destination. We should make that very clear to avoid any... surprises ;)

Reply:
updated